docs/shared/recipes/nx-release/getting-started-with-nx-release.md

```diff
@@ -0,0 +1,273 @@
+# Getting Started with Nx Release
+
+This recipe guides you through versioning packages, generating changelogs, and publishing packages in a JavaScript monorepo with Nx Release.
+
+## Getting set up
+
+### Installing Nx
+
+Ensure that Nx is installed in your monorepo. Check out the [Installation docs](/getting-started/installation) for instructions on created a new Nx workspace or adding Nx to an existing project.
+
+### Adding the JavaScript Plugin
+
+The @nx/js package is required for Nx Release to manage and release JavaScript packages. Add it if it is not already installed:
+
+```shell
+npm install -D @nx/js
+```
+
+### Configure Projects to Release
+
+Nx Release uses Nx's powerful [Project Graph](/core-features/explore-graph) to understand your projects and their dependencies. However, not all projects in your repo need to be released, such as e2e testing projects and some applications.
```

Comment:
Seems strange to declare that not all of a user's projects need to be released, we don't know that 😄 

```suggestion
Nx Release uses Nx's powerful [Project Graph](/core-features/explore-graph) to understand your projects and their dependencies.

If you want to release all of the projects in your workspace, such as when dealing with a series of npm library packages, no configuration is required.

If you have a mixed workspace in which you also have some applications, e2e testing projects or other things you don't want to release, you can configure `nx release` to target only the projects you want to release.
```